src/components/ArticlesList/ArticlesList.js

```diff
@@ -16,8 +16,7 @@ import MoreVertIcon from 'material-ui-icons/MoreVert';
 import EditIcon from 'material-ui-icons/Edit';
 import DeleteForeverIcon from 'material-ui-icons/DeleteForever';
 import InfoIcon from 'material-ui-icons/Info';
-import CheckBoxOutlineBlankIcon from 'material-ui-icons/CheckBoxOutlineBlank';
-import CheckBoxIcon from 'material-ui-icons/CheckBox';
+import Toggle from 'material-ui/Toggle';
```

Comment:
Please use components from `material-ui-next`. `material-ui` is for older library version.

Reply:
OK.